homeassistant/components/unifiprotect/services.yaml

```diff
@@ -84,3 +84,29 @@ set_doorbell_message:
           step: 1
           mode: slider
           unit_of_measurement: minutes
+set_chime_paired_doorbells:
+  name: Set Chime Paired Doorbells
+  description: >
+    Use to set the paired doorbell(s) with a smart chime.
+  fields:
+    entity_id:
+      name: Chime
+      description: The Chimes to link to the doorbells to
+      example: "button.chime_play_chime"
+      required: true
+      selector:
+        entity:
+          integration: unifiprotect
+          domain: button
+          device_class: unifiprotect__chime_button
+    doorbells:
```

Comment:
```suggestion
    doorbell_entity_ids:
```

Reply:
Not entity IDs. It is a target selector. So, it could entity IDs, device IDs or area IDs.

Reply:
https://github.com/home-assistant/core/pull/71874/files#r877674692